service/frontend/clusterRedirectionHandler_test.go

```diff
@@ -708,6 +708,7 @@ func (s *clusterRedirectionHandlerSuite) TestRespondQueryTaskCompleted() {
 	token, err := s.handler.tokenSerializer.SerializeQueryTaskToken(&common.QueryTaskToken{
 		DomainID: s.domainID,
 	})
+	s.Nil(err)
```

Comment:
```suggestion
	s.NoError(err)
```
is probably preferable